aws-ts-static-website/src/lambda-language-redirect.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
as per above, we could likely just have this be inlined as a javascript function whne making the AWS lambda.  Also, it *should* be possible to just make this 'async' instead of the 'callback' form.  But you could definitely just start with directly inlining this first, then trying out the async option afterwards if that's not the case.

Reply:
Done